src/app/components/ExternalLinksCard.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  Title,
+  Bullseye,
+  Split,
+  Flex,
+  FlexItem,
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  Button,
+  Grid,
+  GridItem,
+  CardTitle
+} from '@patternfly/react-core'
+
+
+import redhat_logo from "@app/bgimages/redhat_clear.png";
+import aws_logo from "@app/bgimages/aws_clear.png";
+import azure_logo from "@app/bgimages/azure_clear.png";
+import google_logo from "@app/bgimages/google_clear.png";
+
+
+const providerInfos = [
+  {
+    name: 'aws',
+    text: 'Launch in AWS ->',
```

Comment:
There's an icon in Patternfly for the arrow you might be able to use, search for `fa-arrow-right`:
https://www.patternfly.org/v4/guidelines/icons/#all-icons